src/__tests__/lib/exampleData.js

```diff
@@ -250,12 +250,14 @@ export const stream: Stream = makeStream({
 });
 
 /** A subscription, by default to eg.stream. */
-export const makeSubscription = (args: {| stream?: Stream |} = Object.freeze({})): Subscription => {
-  const { stream: streamInner = stream } = args;
+export const makeSubscription = (
+  args: {| stream?: Stream, in_home_view?: boolean |} = Object.freeze({}),
+): Subscription => {
+  const { stream: streamInner = stream, in_home_view } = args;
   return deepFreeze({
     ...streamInner,
     color: '#123456',
-    in_home_view: true,
+    in_home_view: in_home_view === undefined ? true : in_home_view,
```

Comment:
nit: Could shorten to `in_home_view ?? true`.

This expression has the one difference that it also gives `true` when `in_home_view` is `null`. But `in_home_view` won't be `null` because of its type annotation as an [optional `boolean` property](https://flow.org/en/docs/types/objects/#toc-optional-object-type-properties) of the `args` object.

Reply:
Done! Thanks :)